package.json

```diff
@@ -8,10 +8,12 @@
   "scripts": {
     "release": "changeset publish",
     "build:docs": "pnpm turbo build --filter docs-app",
-    "start": "pnpm start:tests",
-    "start:docs": "pnpm turbo --filter docs-app start",
-    "start:tests": "pnpm turbo --filter test-app start",
-    "start:addon": "pnpm --filter !test-app start --no-watch.clearScreen",
+    "start": "concurrently 'npm:start:*(!only)' --names 'start:'",
+    "start:docs": "pnpm --filter docs-app start",
+    "start:tests": "pnpm --filter test-app start",
+    "start:addon": "pnpm --filter './packages/*' start --no-watch.clearScreen",
+    "start:only:docs": "pnpm --stream --parallel --filter docs-app... start",
```

Comment:
why could I not just run `start:docs` if I wanted to only run the docs? and `start:tests` for just the tests?

Reply:
Because that doesn't run `start:addon` (`start` runs `start:docs`, `start:tests` *and* `start:addon`)...

`start:addon` builds all the `packages/*` and watches them for changes.

`start:only:docs` automatically builds all of the dependencies of docs (thanks to the `...` called out above). We couldn't do the `...` trick in the normal `start` commands because then the packages would end up being built and watched multiple times (once for each consuming app) which causes problems.

It's a shame you can't include comments in `.json` because this seems to be a detail worth preserving somewhere other than a comment on a PR. Maybe I can try and work it into the CONTRIBUTING doc?

Reply:
We can rename `start:only:docs` to `start:docs` but I don't know what we should call the thing currently called `start:docs`... 

It starts the docs app in a way which is only useful if `start:addons` is also running. I guess when I say it like that `start:only:docs` _does_ make sense for that way around (although it really didn't the way I was thinking of it initially). We don't really expect people to run that command anyway (the top level `start` should be enough) so maybe it's ok if the name is a bit confusing...

Maybe we should do:

```
// Starts and watches everything
"start": "concurrently 'npm:start:only:*' --names 'start:'",
// Starts the docs app and builds packages and syncs as appropriate
"start:docs-app": "pnpm --stream --parallel --filter docs-app... start",
// Starts the test app and builds packages and syncs as appropriate
"start:test-app": "pnpm --stream --parallel --filter test-app... start",
// Not intended as public API - maybe we even call them `_start:*` instead of `_start:only:*`?
"start:only:docs-app": "pnpm --filter docs-app start",
"start:only:test-app": "pnpm --filter test-app start",
"start:only:packages": "pnpm --filter './packages/*' start --no-watch.clearScreen",
```

Reply:
I pushed some changes that hopefully help it make more sense?

Reply:
I like that!